app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/ListSkillsResponse.kt

```diff
@@ -1,19 +1,18 @@
 package org.fossasia.susi.ai.rest.responses.susi
 
 import com.google.gson.annotations.Expose
-import com.google.gson.annotations.SerializedName
 
 /**
  *
  * Created by chiragw15 on 16/8/17.
  */
 class ListSkillsResponse {
 
-    @SerializedName("group")
+
     @Expose
     val group: String = "Knowledge"
 
-    @SerializedName("skills")
+
     @Expose
     val skillMap: Map<String, SkillData> = HashMap()
```

Comment:
@S2606 Will this work?

Reply:
@chiragw15 Since the tests are running perfectly i think this should run

Reply:
Or should i change the variable names?

Reply:
@S2606 Which tests? Are talking about UI tests running on circle build? Because those checks does not check all type of responses.

Reply:
@chiragw15 Yes i was talking about those tests